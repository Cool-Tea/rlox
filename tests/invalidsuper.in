class Eclair {
  cook() {
    super.cook();
    print "Pipe full of crème pâtissière.";
  }
}

var obj = Eclair();
obj.cook();