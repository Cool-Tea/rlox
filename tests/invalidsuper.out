Error: Can't use 'super' in a class with no superclass.